pheps/phep-9999.md

```diff
@@ -0,0 +1,171 @@
+      - Silver: Bronze + most recommended fields for HSSI (TBD)
+      - Gold: Silver + all recommended and some optional fields for HSSI (TBD)
+   - Software License: indicates level of software license compliance with the [PyHC Standards Document](https://doi.org/10.5281/zenodo.2529131) 
+   - DOI: indicates whether or not a package's software repository has a DOI (e.g., from Zenodo) 
+   - pip installable?: indicates whether a package is pip installable
+   - conda installable?: indicates whether a package is conda installable
+
+The following table shows the benefits that are associated with each tier:  
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
+| :-: | :----------: | :-----------: | :------------: | :-------------: |
+| Summer School Inclusion \* | Yes | Yes** | No | No |
+| PyHC-top-tier Env Inclusion \* | Yes | Yes ** | No | No | 
+| PyHC-all Env Inclusion \* | Yes | Yes | Yes | No |
+| PyHC-Chat Bot Inclusion \* | Yes | Yes ** | No | No |
+| pyOpenSci Verified Badge | Yes | No | No | No |
+| Standards Compliance Assistance \* | Yes *** | Yes | Yes | Yes |
+| Main Page Listing | Yes | Yes | Yes | No |
+| Secondary Page Listing | No | No | No | Yes |
+| HSSI Inclusion | Yes | Yes | Yes | Yes | 
+| Conference Travel Funding | Yes | Yes | Yes | No |  
+
+\*  = can opt-in or opt-out of this  
+
+\** = conditional upon justification on importance/use of package (e.g. number of users, level of commitment within PyHC activities, or effort made for those items but gold level not yet achieved)  
+
+\*** = conditional upon justification that package is in danger of dropping down to a Silver tier 
+
+Descriptions for each heading are as follows:
+   - Summer School Inclusion: indicates whether a package will be included in summer school teaching materials
+   - PyHC-top-tier Env Inclusion: indicates whether a package will be included within the current PyHC software environment used at the summer school  
+   - PyHC-all Env Inclusion: indicates whether a package will be included within the current PyHC software environment containing all packages Bronze and higher.
+   - PyHC-Chat Bot Inclusion: indicates whether a packages will have up-to-date information included within the ChatGPT4-powered PyHC-Chat bot
+   - pyOpenSci Verified Badge: a badge that shows whether a package has completed the pyOpenSci review process
+   - Standards Compliance Assistance: indicates whether a package will receive extra help and/or funding (dependent on available PyHC resources) for conforming to the PyHC standards
+   - Main Page Listing: indictes whether a package's information will be displayed on a new, main PyHC Project page
+   - Secondary Page Listing: indicates whetheer a package's information will be displayed on a new, secondary PyHC Project page 
+   - HSSI Inclusion: indicates whether a package will be included within the Heliophysics Software Search Interface
```

Comment:
Is this a PyHC benefit or just part of submitting the correct HSSI metadata? Can PyHC put requiremnents on HSSI? Similar for pyOpenSci.

Reply:
PyHC would be requiring PyHC packages at a given level to contribute a given list of metadata fields. PyHC and HSSI are working together on what those lists are. Similarly, PyHC and pyOpenSci are working together to figure out what the requirements should be in that review. Likely needs a PHEP for the pyOpenSci.

Reply:
Basically, what @rebeccaringuette said.

Agreed on the PHEP for pyOpenSci. It'll need community discussion to come to terms with what reqs we want for a package being "PyHC-compliant" during a pyOpenSci review process.

Reply:
So if something is otherwise compliant with pyOpenSci, but isn't doing something right on the PyHC side so it's silver, it won't be included in pyOpenSci? (HSSI inclusion is listed for all tiers so not as relevant).

It seems odd to say a benefit of a particular PyHC tier is being listed *elsewhere*.

Reply:
Part of the pyOpenSci review process will have items or characteristics that are specific to PyHC, similar to that custom pyOpenSci/SunPy review process in development (or now realized, not sure). The point of going through that review process is not to be listed elsewhere, but to decrease the effort needed for PyHC leadership to review those same details of all PyHC packages by hand.
Or did you mean something else?

Reply:
I'm starting to be convinced that the pyOpenSci badge being included with a package indeed should be separate from this PHEP. I think I'm going to remove it, and it can be a separate thing added to a package (but the standard of Gold/Silver for pyOpenSci would still stand, just not then the benefit of the badge being part of this PHEP).

As for the HSSI, I think it still makes sense to include here for PyHC-specific packages, if nothing else than to really encourage packages to get the proper metadata in for a software search interface dedicated to Heliophysics.